cmd/local/install.go

```diff
@@ -108,12 +108,12 @@ func initVarDir() error {
 		return err
 	}
 
-	for _, dir := range []string{"manifests", "storage", "registry"} {
```

Comment:
We don't need the manifests for upgrading?

Reply:
nope, because the `manifests` folder holds the following:
1. k3d cluster configuration file - replaced by an internal struct defined in the cli code
2. helm chart manifest - we we'll no longer support it
3. helm values (in case on `USE_LOCAL_HELM`) - replaced by values inline in the code (the PR you're working on right now IIRC)